AST-1464 - Hide cart drawer (flyout) for mobile & tablet on checkout & cart page. (#3981)
brainstormforce/astra: changelog.txt

- Fix: Off-canvas - Dropdown menu displayed when flyout/fullscreen header type is selected in mobile/tablet customizer.
- Fix: W3 validation - Builder - Account - Duplicate IDs for account icons.
- Fix: Elementor Pro - WooCommerce Grid layout support in WooCommerce Products Archive widget.

> - Fix: WooCommerce - Flyout shopping cart should not be displayed on checkout & cart page for mobile/tablet devices.

Reviewer: ```suggestion
- Fix: WooCommerce - Blank Flyout shopping cart appearing on checkout & cart page for mobile/tablet devices.
```
